tests/python/unittest/test_micro_model_library_format.py

```diff
@@ -411,4 +410,7 @@ def test_export_byoc_c_module():
 
 
 if __name__ == "__main__":
-    sys.exit(pytest.main([__file__] + sys.argv[1:]))
+    import sys
+
+    # sys.exit(pytest.main([__file__] + sys.argv[1:]))
+    test_export_operator_model_library_format()
```

Comment:
We should probably fix this, feel free to do in follow up.

Reply:
Woops, thanks.